### DISCUSSION TITLE
When converting PSD, it will take up a large amount of C disk

### DISCUSSION THREAD

**Discussion by NENCAO** [dlemstra/Magick.NET#1443] [Help] [answered]
### Magick.NET version

q16-x64-13.2.0

### Environment (Operating system, version and so on)

WIN11

### Description

When converting PSD, it will take up a large amount of C disk, even if it will be released later, it will not be all released. This will make users feel panicked.

### Steps to Reproduce

Multiple files PSD -> JPEG

**Comment by dlemstra** [comment, marked as answer]
You will only need C disk space when your are running low on memory. When this library cannot load the image in memory the disk will be used to store the pixels. And those will be cleaned up afterwards. But this sounds like an issue in your program. Are you disposing the image correctly?

**Reply by NENCAO** [reply]
Thank you. I noticed that I used too many threads.
